dist/stat_data.py

```diff
@@ -1006,6 +1006,7 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_eviction_target_page_lt64', 'eviction walk target pages histogram - 32-63'),
     CacheStat('cache_eviction_target_page_reduced', 'eviction walk target pages reduced due to history store cache pressure'),
     CacheStat('cache_eviction_walk_from_root', 'eviction walks started from root of tree'),
+    CacheStat('cache_eviction_walk_random_returns_null_position', 'eviction walks random search fails to locate a page, results in a null position'),
```

Comment:
I'd like a stat that tracks how many retries there have been as well.

Reply:
Updated in [bc745d0](https://github.com/wiredtiger/wiredtiger/pull/10437/commits/bc745d0c0a194de87d585a61032ae0aee2b02918)